guide/creating-your-bot/creating-commands.md

```diff
@@ -45,12 +45,12 @@ pnpm add @discordjs/builders @discordjs/rest discord-api-types
 
 Below is a deployment script you can use. Focus on these variables:
 
-- `clientId`: Your client's id
+- `clientId`: Your bot's client id
```

Comment:
```suggestion
- `clientId`: Your bot's id
```
wouldn't bot's client be simply the bot itself :thinking:

Reply:
There are some edge cases where the application has a different id than the bot account that's attached to it, though as far as i know it's more on the "ancient" side.

Reply:
bot ID should always be application ID, IIRC those were replaced at some point

Reply:
@almostSouji no, they were not, or at least not fully if that ever happened. The only bot i know that has such "issue" still has a different ID in the invite link than the bot account it uses. It doesn't even "redirect", and just gives you "unknown application".

Admittedly though, with the addition of the "Add to Server" button (which bot in question does use), i don't think this will be anyhow common problem (on top of the fact that you had to know how to "make" a bot invite link in order to actually bump into it in the first place).